flypg/launcher.go

```diff
@@ -155,6 +162,7 @@ func (l *Launcher) LaunchMachinesPostgres(ctx context.Context, config *CreateClu
 						},
 					},
 					Concurrency: concurrency,
+					Autostart:   &autoStart,
```

Comment:
AFAIK, you also need to remove `machineConf.DisableMachineAutostart`, otherwise `flaps` always overwrites service meta tags and will clear `autostart=true`.

Reply:
`DisableMachineAutostart` is marked deprecated in favor of the service.Autostart field. I think this was changed last week

Reply:
Good point. Do not use `DisableMachineAutostart`, it is deprecated.

Reply:
Yeah, but it's being set in this file below (line 212) and AFAIK `flaps` uses it if it's set ignoring the rest of the `autostart`/`autostop` parameters.